app/prices.js

```diff
@@ -10,6 +11,9 @@ const socket = require('lib/ptf-socket');
 
 const handlerManager = require('app/handler-manager');
 
+// Max age of a price in seconds
+const maxPriceAge = parseInt(process.env.MAX_PRICE_AGE) || 8 * 60 * 60;
```

Comment:
I'd set the default value to 0 (or 4 hours, since pricestf updates each price every ~4 hours), since this will only affect a few people with really big pricelists

Reply:
It aims for 4 hours but when people like @ZeusJunior spams the API and because of the backpack.tf rate limiting then it ends up being a lot slower.

Reply:
ITS NOT THAT MANY

Reply:
My point being, that most people will set it to as low as possible, so it would make sense to use that as default